include/dpp/permissions.h

```diff
@@ -146,6 +146,25 @@ class DPP_EXPORT permission {
 		return (value & (0 | ... | values)) == (0 | ... | values);
 	}
 
+	/**
+	 * @brief Check for permission flags set. It uses the Bitwise AND operator
+	 * @tparam T one or more uint64_t permission bits
+	 * @param values The permissions (from dpp::permissions) to check for
+	 *
+	 * **Example:**
+	 *
+	 * ```cpp
+	 * bool is_mod = permission.has(dpp::p_administrator, dpp::p_ban_members);
```

Comment:
is this meant to be `has_eny()`? not just `has()`?

Reply:
permission can take a list like this, if I remember correctly

Reply:
> permission can take a list like this, if I remember correctly

`has` is check that you have all listed permissions, but `has(dpp::p_adminstrator | dpp::p_ban_members)` does